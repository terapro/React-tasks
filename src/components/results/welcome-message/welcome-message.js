import React from 'react';

export const WelcomeMessage = () => (
  <div className='welcome-message'>
    Type the film name to start search
  </div>

);
