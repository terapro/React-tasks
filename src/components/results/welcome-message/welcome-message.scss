.welcome-message {
  min-height: 110px;
  width: inherit;
  text-align: center;
  font-size: 40px;
  padding-top: 200px;
}

.welcome-message span {
  color: #0056b3;
}