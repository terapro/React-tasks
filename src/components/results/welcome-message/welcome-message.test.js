import React from 'react';
import {WelcomeMessage} from "src/components/results/welcome-message/welcome-message";
import {shallow} from 'enzyme';

describe('<WelcomeMessage />', () => {
  describe('Rendering', () => {
    it('should render the component', () => {
      const wrapper = shallow(
        <WelcomeMessage />
      );
      expect(wrapper).toMatchSnapshot();
    });
  });
});
