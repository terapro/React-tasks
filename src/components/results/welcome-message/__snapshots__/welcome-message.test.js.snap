// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeMessage /> Rendering should render the component 1`] = `
<div
  className="welcome-message"
>
  Type the film name to start search
</div>
`;
